src/SparseLRModel.h

```diff
@@ -172,6 +182,9 @@ class SparseLRModel : public CirrusModel {
     //mutable std::unordered_map<uint32_t, FEATURE_TYPE> weights_sparse_;
     mutable std::vector<FEATURE_TYPE> weights_sparse_;
 
+    FEATURE_TYPE momentum_avg = 0.0;
+    double nesterov_beta = 0.9;
```

Comment:
This value is never changed. Shouldn't this be a configuration parameter? If so, should add support for this in Configuration.h/cpp.